creature/BoltNineHomebrew; The King's Army - Complete Monster Pack.json

```diff
@@ -0,0 +1,2563 @@
+				],
+				"images": [
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "https://i.imgur.com/U4AsxU7.png"
+						}
+					}
+				]
+			},
+			"tokenUrl": "https://i.imgur.com/U4AsxU7.png",
+			"shortName": "War Mage",
+			"languages": [
+				"Common",
+				"Elvish"
+			],
+			"languageTags": [
+				"C",
+				"E"
+			],
+			"action": [
+				{
+					"name": "Quarterstaff (One-Handed)",
+					"entries": [
+						"{@atk mw} {@hit 1} to hit, reach 5 ft., one target. {@h}2 ({@damage 1d6 + 1}) bludgeoning damage."
+					]
+				},
+				{
+					"name": "Quarterstaff (Two-Handed)",
+					"entries": [
+						"{@atk mw} {@hit 1} to hit, reach 5 ft., one target. {@h}3 ({@damage 1d8 + 1}) bludgeoning damage."
+					]
+				},
+				{
+					"name": "Mob Attack",
+					"entries": [
+						"5e DMG: page 250 as a complete mass combat system",
+						"The 5e DMG has a short section on “handling mobs:” it has a chart for approximating, out of a group of attacking monsters, how many monsters hit.",
+						"Subtract attacker’s hit bonus from the target’s AC. Cross-index that number on the chart. If the number is 1-5, all the attackers hit; if it’s 6-12, 1/2 of them hit; etc., up to 1 in 20 of the attackers hitting on a 20. "
+					]
+				}
+			],
+			"damageTags": [
+				"B"
+			],
+			"miscTags": [
+				"MW"
+			],
+			"isNpc": true,
+			"environment": [
+				"forest"
+			],
+			"trait": [
+				{
+					"name": "INFANTRY",
+					"entries": [
+						"An army is nothing without its foot soldiers. Either in small groups or large armies, men and women travel the King’s lands, enforcing laws and maintaining peace."
+					]
+				},
+				{
+					"name": "Pieces of Whole",
+					"entries": [
+						"The small units of the King’s army occasionally come together under the leadership of a high ranking military official. However, due to the sheer number of troops, each group is still under the authority of the sergeant."
+					]
+				},
+				{
+					"name": "Modular Units",
+					"entries": [
+						"A unit of the King’s Army is composed of 4 to 5 infantryman, 2 to 3 spearmen, 3 to 4 archers, 1 to 2 cavalry scout, and 1 war mage under the leadership of a sergeant. This is the smallest unit of troops that the King deploys into his lands."
+					]
+				}
+			],
+			"skill": {
+				"arcana": "+6"
+			},
+			"spellcasting": [
+				{
+					"name": "Spellcasting",
+					"headerEntries": [
+						"Spellcasting Ability=INT",
+						"Global Magic Attack Modifier=5",
+						"Magic Caster Level=5",
+						"Spell Save DC Mod=13",
+						"The war mage is a 5th level spellcaster. His spellcasting ability is Intelligence (spell save {@dc 13}, +5 to hit with spell attacks). The war mage has the following wizard spells prepared:",
+						"Cantrips (at will): firebolt, mage hand, message,prestidigitation",
```

Comment:
Another duplicated list

Reply:
Hi removed duplicate spell list, when i have time i will try to beautify the fluffy description ^^